.claude/commands/process-renovate-pr.md

```
@@ -0,0 +1,216 @@
+   {command}
+   ```
+
+2. **{Step 2}:**
+   ```bash
+   {command}
+   ```
+
+{Continue with all testing steps}
+
+**Expected behavior:** {What should happen when tests pass}
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)